main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
```

Comment:
Nice work! I wonder if we could use a hash table to store all the inputs as key/value pairs and use them for the print statements.